src/libs/actions/Timing.ts

```diff
@@ -39,8 +39,8 @@ function end(eventName: string, secondaryName = '', maxExecutionTime = 0) {
         return;
     }
 
+    const {startTime, shouldUseFirebase} = timestampData[eventName];
```

Comment:
Hm... What if `timestampData[eventName]` is `undefined`? We'll still get an error here. Can't we just do:
```
    const {startTime, shouldUseFirebase} = timestampData[eventName] || {};
```
instead inside the callback?

Reply:
Probably, we need to set a default value for `startTime` as well?

Reply:
Ah yeah I thought the same but we're already checking if the value exists in the condition above, also we're substracting `startTime` which also won't be ideal. `timestampData[eventName]` exists but it's deleted in another .end call